TourDeOpole/TourDeOpole/Views/PlaceDetailsView.xaml

```diff
@@ -1,14 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" 
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" 
+             xmlns:viewmodels="clr-namespace:TourDeOpole.ViewModels"
+             x:DataType="viewmodels:PlaceDetailsViewModel"
              x:Class="TourDeOpole.Views.PlaceDetailsView"
+             Shell.NavBarIsVisible="False"
```

Comment:
Tutaj jak dasz True to pojawi ci się pasek i z niego korzystajmy nie z takiego zrobionego samemu bo potem ciężko będzie z nawigacją. Jedyny problem będzie z tym obrazkiem udostępniania, to się da zrobić chyba z jakiejś paczki ale możesz na razie to olać.